### PR TITLE
Update low-code-cdk-overview.md

### PR DESCRIPTION
**Pull request by Jeroendevr**
## What
For those not using the Builder UI, you might skip the [section](https://docs.airbyte.com/connector-development/config-based/low-code-cdk-overview#connector-builder-ui) where the YAML file is called the Connector Manifest. So as a reader you only read the section [Configuring the YAML file](https://docs.airbyte.com/connector-development/config-based/low-code-cdk-overview#configuring-the-yaml-file) 
Coming from the tutorial I had to search where the Connector Manifest is discussed in more detail. 

### Rewording
The sentence

> The low-code framework involves editing the Connector Manifest a boilerplate YAML file. The general structure of the YAML file is as follows:

Reads as if the Connector Manifest is _boilerplate_, perhaps its more clear to remove _boilerplate_?

> The low-code framework involves editing the Connector Manifest a YAML file. The general structure of the YAML file is as follows:

## How
N.A.

## Review guide
N.A

## User Impact
Explicit naming that the _boilerplate yaml file_ is the Connector Manifest

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Review by marcosmarxm — COMMENTED** [review]
Thanks @Jeroendevr, @Adorism can you take a look into this suggestion?

File: docs/connector-development/config-based/low-code-cdk-overview.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ We recommend iterating on this YAML file is via the [connector builder UI](https
 
 ## Configuring the YAML file
 
-The low-code framework involves editing a boilerplate YAML file. The general structure of the YAML file is as follows:
+The low-code framework involves editing the Connector Manifest a boilerplate YAML file. The general structure of the YAML file is as follows:
```
Comment (Adorism) [inline docs/connector-development/config-based/low-code-cdk-overview.md:92]:
```suggestion
The low-code framework involves editing the Connector Manifest, which is a boilerplate YAML file. The general structure of the YAML file is as follows:
```

**Review by Adorism — APPROVED** [review]
Looks good, made an edit suggestion for grammatical clarity.

**Comment by Jeroendevr** [comment]
@Adorism , thank you for approving. Question about this PR. I have updated a couple of times the branch as I thought this was keeping me from merging this PR. But it waits on `Run connectors full CI on fork` is it usual that it takes two days or is something else going on? 
Thanks,

**Comment by marcosmarxm** [comment]
Hello your feedback matters a lot to us. Can you spare just 3 minutes to complete a [survey](https://form.typeform.com/to/DOBqejte)? We’re dedicated to making the contributor experience even better, and your input is key to achieving excellence. Thank you for helping us improve!

**Pull request merged**